### PR TITLE
minor fixes in Parameter spec

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *minor fixes in Parameter spec*

### Tickets:
 - *52870*

### CODE REVIEW

File: docs/ops/infrastructure/Parameter_1.md [hunk 1]
```diff
@@ -11,18 +11,27 @@
 * *element_type*
 
   * **Description**: the type of element of output tensor
-  * **Range of values**: u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
-  * **Type**: string
+  * **Range of values**: u1, u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
```
Comment (jdanieck) [inline docs/ops/infrastructure/Parameter_1.md:14]: What about `u4` & `i4` ?

File: docs/ops/infrastructure/Parameter_1.md [hunk 2]
```diff
@@ -11,18 +11,27 @@
...
+  * **Range of values**: list of non-negative integers, empty list is allowed, which means 0D or scalar tensor
+  * **Type**: `int[]`
   * **Default value**: None
-  * **Required**: *Yes*
+  * **Required**: *yes*
+
+
+**Outputs**
+
+* **1**: Output tensor of type *T*.
```
Comment (jdanieck) [inline docs/ops/infrastructure/Parameter_1.md:30]: Missing info about output shape.

File: docs/ops/infrastructure/Parameter_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/infrastructure/Parameter_1.md]: add

File: docs/ops/infrastructure/Parameter_1.md (hunk 2 above)
Comment (pelszkow) [reply docs/ops/infrastructure/Parameter_1.md]: fixed

**Review by jdanieck — APPROVED** [review]

**Review by ggalieroc — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**